src/cpp/session/modules/SessionProfiler.R

```diff
@@ -90,7 +90,7 @@
 {
    tryCatch({
       resources <- .rs.profileResources()
-      htmlFile <- normalizePath(tempfile(fileext = ".html", tmpdir = resources$tempPath), winslash = "/")
+      suppressWarnings(htmlFile <- normalizePath(tempfile(fileext = ".html", tmpdir = resources$tempPath), winslash = "/"))
```

Comment:
Interesting -- for some reason I was under the impression this wasn't true. Sorry, and thanks for double checking!